app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
-end
+  def full_title(page_title = '')
+  	page_title.empty? ? "AID-Interview" :  page_title + " | " + "AID-Interview"
```

Comment:
Can be done in a better way.

hint: 
```ruby
def full_title(page_title = 'nil)
```
then you could use some [array magic](http://docs.ruby-lang.org/en/2.0.0/Array.html) to build a title